tests/FixerConfiguration/AliasedFixerOptionTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\FixerConfiguration;
+
+use PhpCsFixer\FixerConfiguration\AliasedFixerOption;
+use PhpCsFixer\FixerConfiguration\FixerOption;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\FixerConfiguration\AliasedFixerOption
+ */
+final class AliasedFixerOptionTest extends TestCase
+{
+    public function testGetName()
+    {
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
+        $this->assertSame('foo', $option->getName());
+    }
+
+    public function testGetDescription()
+    {
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
+        $this->assertSame('Bar.', $option->getDescription());
+    }
+
+    public function testHasDefault()
+    {
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
+        $this->assertFalse($option->hasDefault());
+
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.', false, 'baz'), 'baz');
+        $this->assertTrue($option->hasDefault());
+    }
+
+    public function testGetDefault()
+    {
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.', false, 'baz'), 'baz');
+        $this->assertSame('baz', $option->getDefault());
+    }
+
+    public function testGetUndefinedDefault()
+    {
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
+
+        $this->setExpectedException('LogicException', 'No default value defined.');
+        $option->getDefault();
+    }
+
+    public function testGetAllowedTypes()
+    {
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.'), 'baz');
+        $this->assertNull($option->getAllowedTypes());
+
+        $option = new AliasedFixerOption(new FixerOption('foo', 'Bar.', true, null, array('bool')), 'baz');
```

Comment:
Please use `@dataProvider`s to extract multiple cases from the test implementation itself (same for other tests).

Reply:
As these tests are copied from `FixerOptionTest`, shall I change the tests there as well? Or shall I open a separate PR for that?

Reply:
I'd say separate PR :)